### PR TITLE
Project Page Detail Part 3

### PR DESCRIPTION
**Pull request by ketphan02**
Dependency: #46 and #47 

This PR adds the Project Set page

### CODE REVIEW

File: src/app/(app)/project-sets/columns.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+"use client"
+
+import {type ColumnDef} from "@tanstack/table-core"
+import {Checkbox} from "@/components/ui/checkbox"
+import {DataTableColumnHeader} from "@/components/ui/data-table-column-header"
+import {type ProjectSet} from "@/_temp_types/projects"
+
+export const columns: ColumnDef<ProjectSet>[] = [
+  {
+    id: "select",
```
Comment (Opeyem1a) [inline src/app/(app)/project-sets/columns.tsx:10]: what are the bulk actions we currently envision for project sets?

**Review by Opeyem1a — COMMENTED** [review]
This PR is probably fine, some issues I will address sometime later.

I'm gatekeeping it without approval until you fix the dependent branches though

File: src/app/(app)/project-sets/columns.tsx (hunk 1 above)
Comment (ketphan02) [reply src/app/(app)/project-sets/columns.tsx:10]: For now just deleting, but we haven't implemented that yet

File: src/components/ui/use-toast.ts
Comment (Opeyem1a) [inline src/components/ui/use-toast.ts:1]: this is a duplicate file

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

**Review by Opeyem1a — APPROVED** [review]

**Pull request merged**